test/unittests/api/v0.0.1/candidates/Visit_Test.php

```diff
@@ -89,6 +89,7 @@ function testPutInvalidBattery() {
         $this->assertEquals($API->Headers, ["HTTP/1.1 400 Bad Request"]);
     }
     function testPostValidVisit() {
+        \User::singleton("admin");
```

Comment:
There also is no way to do a `User::logOut()` or similar which immediately forces the `User::singleton()` call to return null, effectively logging them out.

I can do the following,
A) Close this pull request because we can just abandon this 0.X API thing, right?
B) Refactor all of Loris, which I can't because that's not allowed, right?
C) Comment out the tests with a reason, which I also can't because that's not allowed, right?